cob_hardware_config/cob4-4/config/Elmo.dcf

```diff
@@ -1,4502 +0,0 @@
-[FileInfo]
```

Comment:
@ipa-mdl 
please verify the dcf contains your latest valid configuration/changes/values